pkg/node/node_address.go

```diff
@@ -501,7 +511,7 @@ func getCiliumHostIPsFromNetDev(devName string) (ipv4GW, ipv6Router net.IP) {
 			}
 		}
 	}
-	return nil, nil
```

Comment:
Did this never work?

Reply:
@raybejjani The function? Perhaps not as the `node_config.h` was always available

Reply:
Based on the unit test failing all of a sudden I'm confident that device restore didn't work before. The unit test can't read the node_config.h due to permissions which is why it broke. Regular environments are likely always capable of reading node_config.h